### PR TITLE
feat: rather than a force-prune-all-images...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

feat: rather than a force-prune-all-images (which prevents the mm notifier from working), just remove any existing theia images from local cache

Change-Id: I838d23a8b66a7d460e83b70eb5bdce94abbf0819
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by azatsarynnyy — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -70,9 +70,13 @@ jobs:
         export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
         export AKAMAI_CHE_AUTH=${{ secrets.AKAMAI_CHE_AUTH }}
         ./make-release.sh --version ${{ github.event.inputs.version }}
+    - name: Clean image cache
```
Comment (benoitf) [inline .github/workflows/release.yml:73]: hello, I think it won't remove anything

Comment (nickboldt) [reply .github/workflows/release.yml:73]: so you'd suggest we do `prune -f` instead ?

**Comment by benoitf** [comment]
should probably move the notification to it's own job so it stars with fresh docker containers

**Comment by nickboldt** [comment]
replaced by https://github.com/eclipse/che-theia/pull/1029

**Comment by azatsarynnyy** [comment]
@nickboldt could you please delete the branch if it's not needed anymore

**Pull request closed without merging**